include/D2D/lambertScanner.hpp

```diff
@@ -52,22 +52,26 @@ struct LambertScannerInput
      * Constructs data struct based on verified input parameters.
      *
      * @sa checkLambertScannerInput, executeLambertScanner
-     * @param[in] aCatalogPath          Path to TLE catalog
-     * @param[in] aDatabasePath         Path to SQLite database
-     * @param[in] aDepartureEpoch       Departure epoch for all transfers
-     * @param[in] aTimeOfFlightMinimum  Minimum time-of-flight [s]
-     * @param[in] aTimeOfFlightMaximum  Maximum time-of-flight [s]
-     * @param[in] someTimeOfFlightSteps Number of steps to take in time-of-flight grid
-     * @param[in] aTimeOfFlightStepSize Time-of-flight step size (derived parameter) [s]
-     * @param[in] progradeFlag          Flag indicating if prograde transfer should be computed
-     *                                  (false = retrograde)
-     * @param[in] aRevolutionsMaximum   Maximum number of revolutions
-     * @param[in] aShortlistLength      Number of transfers to include in shortlist
-     * @param[in] aShortlistPath        Path to shortlist file
+     * @param[in] aCatalogPath             Path to TLE catalog
+     * @param[in] aDatabasePath            Path to SQLite database
+     * @param[in] aDepartureEpochInitial   Departure epoch grid initial epoch
+     * @param[in] aDepartureEpochSteps     Number of steps to take in departure epoch grid
+     * @param[in] aDepartureEpochStepSize  Departure epoch grid step size (derived parameter) [s]
+     * @param[in] aTimeOfFlightMinimum     Minimum time-of-flight [s]
+     * @param[in] aTimeOfFlightMaximum     Maximum time-of-flight [s]
+     * @param[in] someTimeOfFlightSteps    Number of steps to take in time-of-flight grid
+     * @param[in] aTimeOfFlightStepSize    Time-of-flight step size (derived parameter) [s]
+     * @param[in] progradeFlag             Flag indicating if prograde transfer should be computed
+     *                                     (false = retrograde)
+     * @param[in] aRevolutionsMaximum      Maximum number of revolutions
+     * @param[in] aShortlistLength         Number of transfers to include in shortlist
+     * @param[in] aShortlistPath           Path to shortlist file
      */
     LambertScannerInput( const std::string& aCatalogPath,
                          const std::string& aDatabasePath,
-                         const DateTime&    aDepartureEpoch,
+                         const DateTime&    aDepartureEpochInitial,
+                         const double       aDepartureEpochSteps,
```

Comment:
@ennehekma To be consistent with the time-of-flight variable, can you change this to:
```
someDepartureEpochSteps
```